src/redux/sagas/actions/mintTokens.ts

```diff
@@ -113,21 +120,39 @@ function* createMintTokensAction({
     yield put(transactionReady(claimColonyFunds.id));
     yield takeFrom(claimColonyFunds.channel, ActionTypes.TRANSACTION_SUCCEEDED);
 
-    // if (annotationMessage) {
-    //   yield put(transactionPending(annotateMintTokens.id));
+    if (annotationMessage) {
+      yield put(transactionPending(annotateMintTokens.id));
+
+      // const ipfsHash = yield call(uploadIfpsAnnotation, annotationMessage);
+
+      yield apolloClient.mutate<
+        CreateAnnotationMutation,
+        CreateAnnotationMutationVariables
+      >({
+        mutation: CreateAnnotationDocument,
+        variables: {
+          input: {
+            message: annotationMessage,
```

Comment:
I believe this is error is showing up as the `generated.ts` is different from the schema